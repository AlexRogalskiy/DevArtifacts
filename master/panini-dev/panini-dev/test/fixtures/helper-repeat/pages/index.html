{{#repeat 5}}
<p>Repeat {{@index}}!</p>
{{/repeat}}
