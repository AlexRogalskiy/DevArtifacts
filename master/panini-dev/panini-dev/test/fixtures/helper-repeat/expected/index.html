<html>
  <body>
    <p>Repeat 0!</p>
    <p>Repeat 1!</p>
    <p>Repeat 2!</p>
    <p>Repeat 3!</p>
    <p>Repeat 4!</p>
    
  </body>
</html>
